GCRCatalogs/alphaq.py

```diff
@@ -85,6 +110,42 @@ def native_quantity_getter(native_quantity):
                 return fh['galaxyProperties/{}'.format(native_quantity)].value
             yield native_quantity_getter
 
+    def _get_native_quantity_info_dict(self,quantity,default=None):
+        with h5py.File(self._file,'r') as fh:
+            if 'galaxyProperties/'+quantity not in fh:
+                return default
+            else:
+                info_dict = dict()
+                for key in fh['galaxyProperties/'+quantity].attrs.keys():
+                    info_dict[key] = fh['galaxyProperties/'+quantity].attrs[key]
+                return info_dict
+    
+    def _get_quantity_info_dict(self, quantity, default=None):
+        return default
+        #TODO needs some fixing
+        print "in get quantity"
```

Comment:
Note that the code is now required to be 2-3 compatible. Please use print function.